recipes/pbccs/meta.yaml

```diff
@@ -9,10 +9,12 @@ package:
 source:
   url: https://github.com/PacificBiosciences/unanimity/releases/download/v{{ version }}/ccs # [linux]
```

Comment:
Just remove the `# [linux]` here and the lines below. You are skipping OSX below.

Reply:
Thanks, that did it. Is this a change in behaviour from CB2?